### DISCUSSION TITLE
Need optimization

### DISCUSSION THREAD

**Discussion by iamzaidsoomro** [lowlighter/metrics#830] [Ideas]
Whenever I try to open profiles that use metrics, their readme file doesn't load properly in most cases.

**Comment by lowlighter** [comment]
Hi 👋 !

Well, it mostly depends on how this action has been configured by users.

Using a lot of plugins (especially the ones using images) will eventually make the render heavier and slower but there's nothing that can be done about it, except using less plugins

If you're talking about users who directly uses the web instance, then it's normal since the VPS hosting it has probably not enough resources to fulfill all incoming requests
